### PR TITLE
Adds lowMem option from older dep state

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,40 +5,6 @@ checkout-linux: &checkout-linux
     at: /root
 
 jobs:
```
Comment (andrewxhill) [inline .circleci/config.yml:7]: Moved the unit tests to actions for now

File: mobile/mobile_test.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func TestStart(t *testing.T) {
-	port, err := Start("/tmp/mobile-ipfs-lite", false)
+	port, err := Start("/tmp/mobile-ipfs-lite", false, true)
```
Comment (andrewxhill) [inline mobile/mobile_test.go:9]: runs lowmem mode in the mobile tests

File: util/util.go [hunk 3]
```diff
@@ -58,7 +60,11 @@ func NewPeerManager(ctx context.Context, datastorePath string, port int, debug b
 	var peerManager = StandardManager{}
 
 	var err error
-	peerManager.datastore, err = ipfslite.BadgerDatastore(datastorePath)
+	opts := badger.DefaultOptions
+	if lowMem {
+		opts.TableLoadingMode = options.FileIO
+	}
+	peerManager.datastore, err = badger.NewDatastore(datastorePath, &opts)
```
Comment (andrewxhill) [inline util/util.go:67]: this is the major change of the pr

**Review by carsonfarmer — APPROVED** [review]
Nice, LGTM @andrewxhill

**Review by asutula — APPROVED** [review]

**Pull request merged**